seaborn/categorical.py

```diff
@@ -1672,7 +1672,7 @@ class _PointPlotter(_CategoricalStatPlotter):
     def __init__(self, x, y, hue, data, order, hue_order,
                  estimator, ci, n_boot, units,
                  markers, linestyles, dodge, join, scale,
-                 orient, color, palette, errwidth=None, capsize=None):
+                 orient, color, palette, errwidth=None, capsize=None, label=None):
```

Comment:
If you look at the logs, you will see that the build fails because this line is too long.

```
seaborn/categorical.py:1675:80: E501 line too long (82 > 79 characters)
make: *** [lint] Error 1
The command "make lint" exited with 2.
```

Reply:
Yes, thank you for reply....